docs/api.rst

```diff
@@ -25,6 +25,5 @@
 API Docs
 ========
 
-invenio_deposit
----------------
-
+.. automodule:: invenio_deposit.api
```

Comment:
Yes. We need to document it in invenio-records-rest and upload new docs on pythonhosted.